doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+			constructor (storage) {
+		
+			}
+		}
+		
+		export var myValue = value;
+	</source-code>
+</code-listing>
+
+As long as you define your exports semantically, Aurelia doesn't need to register or define them in a specific way ("factory" vs "service" vs "value"). Everything is an exported module; everything is generic, and modules can be imported and then injected using Aurelia's dependency injection.
+
+For more information on dependency injection, see the documentation here.
+
+### Migrating your ViewModels
+
+Fourth, we'll migrate our viewmodels.
+
+In Angular, a controller must be registered with a specific name, using the `angular` global object.
+
+<code-listing heading="Angular ViewModel Definitions">
+	<source-code lang="ES 2015/2016">
+		function Users ($http, $scope) {
+			...
+		}
+		
+		angular.controller('users', ['$scope', '$http', Users]);
+	</source-code>
+</code-listing>
+
+Using Aurelia, any object can act as a viewmodel without being explicitly registered, as long as that object is exported as a module. Essentially, this simply means removing the Angular boilerplate registration code. So, essentially, you would have this:
+
+<code-listing heading="Aurelia ViewModel Definitions">
+	<source-code lang="ES 2015/2016">
+		function Users (httpClient) {
+			...
+		}
+	</source-code>
+</code-listing>
+
+> Note: Dependency Injection
+> Angular was able to inject Angular-specific dependencies based on the argument name (such as `$scope` and `$http`) through its service locator. Because Aurelia does not couple your application to Aurelia-specific implementations, all dependencies must be explicitly injected.
+
+#### Routing
+
+Aurelia supports client-side routing in the browser. The main difference between Angular's router and Aurelia's router is in the configuration; the Aurelia router can be configured outside of startup, and it can be configured asynchronously. Finally, the Aurelia router supports nested routers (also known as child routers) as well as multiple viewports.
+
+An Angular router configuration, in startup, may look like this:
+
+Alternatively, an Aurelia router is configured inside the viewmodel that uses it. Thus, Aurelia configuration may look like this:
+
+<code-listing heading="Aurelia Router Configuration">
+	<source-code lang="ES 2015/2016">
+		export class App {
+		  configureRouter(config, router) {
+		    config.title = 'Aurelia';
+		    config.map([
+		      { route: ['', 'welcome'], name: 'welcome',      moduleId: 'welcome',      nav: true, title: 'Welcome' },
+		      { route: 'users',         name: 'users',        moduleId: 'users',        nav: true, title: 'Users' }
+		    ]);
+		
+		    this.router = router;
+		  }
+		}
+	</source-code>
+</code-listing>
+
+> Note: Router Migration
+> Migration to Aurelia will be easier if your application uses Angular's new router; this router shares many similarities with the router that Aurelia uses, and the configuration will be easier to migrate.
+
```

Comment:
@EisenbergEffect, would it be too much to montion that the mastermind of Aurelia also created Angular's new router: https://github.com/angular/router/graphs/contributors :P